include/alpaka/dev/cpu/SysInfo.hpp

```diff
@@ -123,6 +123,26 @@ namespace alpaka
                 return std::string("unknown");
 #endif
             }
+
+            //! \return Pagesize in bytes used by the system.
+            inline size_t getPageSize()
+            {
+#if BOOST_OS_WINDOWS || BOOST_OS_CYGWIN
+                SYSTEM_INFO si;
+                GetSystemInfo(&si);
+                return si.dwPageSize;
+#elif BOOST_OS_UNIX || BOOST_OS_MACOS
```

Comment:
Shouldn't `BOOST_OS_UNIX` also work for macOS?

Reply:
I do not know I have no MAC and other code snippeds in alpaka use the same combination.